pkg/rbd/rbd_util.go

```diff
@@ -479,25 +469,18 @@ func genVolFromVolumeOptions(ctx context.Context, volOptions, credentials map[st
 		}
 	}
 
-	rbdVol.ImageFormat, ok = volOptions["imageFormat"]
-	if !ok {
-		rbdVol.ImageFormat = rbdImageFormat2
-	}
-
-	if rbdVol.ImageFormat == rbdImageFormat2 {
-		// if no image features is provided, it results in empty string
-		// which disable all RBD image format 2 features as we expected
-		imageFeatures, found := volOptions["imageFeatures"]
-		if found {
-			arr := strings.Split(imageFeatures, ",")
-			for _, f := range arr {
-				if !supportedFeatures.Has(f) {
-					return nil, fmt.Errorf("invalid feature %q for volume csi-rbdplugin, supported"+
-						" features are: %v", f, supportedFeatures)
-				}
+	// if no image features is provided, it results in empty string
```

Comment:
Nit: comment is not applicable anymore

Reply:
removed comment

Reply:
Sorry, I misread that comment when I was speeding through the review (thought it was talking about the image format). It is still applicable here.

Reply:
added back :)